lib/absinthe/schema/error.ex

```diff
@@ -10,7 +10,10 @@ defmodule Absinthe.Schema.Error do
       |> Enum.map(fn %{message: message, locations: locations} ->
         locations =
           locations
-          |> Enum.map(&"#{&1.file}:#{&1.line}")
+          |> Enum.map(fn
+            %{line: line, file: file} -> "#{file}:#{line}"
+            %{column: column, line: line} -> "Column #{column}, Line #{line}"
```

Comment:
This is needed since the phase error doesn't reference a file & line in SDL

Reply:
Is it not possible to know the file?

Reply:
The SDL parser seems to take the SDL string as input, it didn't have any info about the file it was contained in...